### DISCUSSION TITLE
Padding after adding color

### DISCUSSION THREAD

**Discussion by a-khushal** [LazyVim/LazyVim#4495] [Q&A]
![Screenshot from 2024-09-26 20-35-36](https://github.com/user-attachments/assets/f65263bb-869e-4a36-9b33-f5de8cb5499a)
does anyone know what's wrong? 
There's a padding that's been added towards the end of the terminal screen

**Comment by dpetka2001** [comment]
That's just because of the fact that Neovim is a terminal application and terminals perceive space in the context of cells. So, if the height/width of your terminal is not divisible with the size of the cell you'll observe these paddings. Try to experiment with the borders padding of your terminal and maybe also your line height.

**Reply by a-khushal** [reply]
Yep, understood, I changed by terminal preferences and now it looks good. Thankyou!
